code/datums/components/team_monitor.dm

```diff
@@ -49,7 +49,7 @@ GLOBAL_LIST_EMPTY(tracker_beacons)
 
 /proc/get_all_beacons_on_frequency(frequency, base_frequency)
 	if(!frequency)
-		return list()
+		return GLOB.tracker_beacons["[base_frequency]-GLOB"]
```

Comment:
Why this change?

Reply:
If no frequency was provided then global signals wouldn't be displayed on the monitor HUD